package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "typescript",
+  "version": "1.0.0",
+  "type": "module",
+  "description": "A repository for All algorithms implemented in Typescript (for educational purposes only)",
+  "main": "",
+  "devDependencies": {
+    "@babel/core": "^7.19.1",
+    "@babel/preset-env": "^7.19.1",
+    "@babel/preset-typescript": "^7.18.6",
+    "@types/jest": "^29.0.3",
+    "babel-jest": "^29.0.3",
+    "husky": "^8.0.1",
+    "jest": "^29.0.3",
+    "ts-jest": "^29.0.2",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.8.3"
+  },
+  "scripts": {
+    "test": "jest --no-cache",
+    "style": "standard",
+    "prepare": "husky install"
+  },
+  "author": "TheAlgorithms",
+  "license": "GPL-3.0"
```

This is wrong. We're using the MIT license.
```suggestion
  "license": "MIT"
```

---

Ooh... I will update it.